app/models/GetPensionChargesRequestModel.scala

```diff
@@ -29,7 +30,24 @@ object Charge {
 
 case class PensionSchemeOverseasTransfers(overseasSchemeProvider: Seq[OverseasSchemeProvider],
                                           transferCharge: BigDecimal,
-                                          transferChargeTaxPaid: BigDecimal)
+                                          transferChargeTaxPaid: BigDecimal) {
+  def isEmpty: Boolean = this.overseasSchemeProvider.isEmpty && transferCharge != 0 && transferChargeTaxPaid != 0
+
+  def toTransfersIntoOverseasPensions(
```

Comment:
I would add a unit test, as the logic is pretty not standard. Using zeros for booleans